sortinghat/api.py

```diff
@@ -446,6 +448,23 @@ def edit_profile(db, uuid, **kwargs):
         if 'email' in kwargs:
             profile.email = to_none_if_empty(kwargs['email'])
 
+        if 'gender' in kwargs:
+            profile.gender = to_none_if_empty(kwargs['gender'])
+
+            if not profile.gender:
+                gender_acc = None
+            else:
+                gender_acc = kwargs.get('gender_acc', 100)
+
+                if type(gender_acc) != int:
+                    raise WrappedValueError('gender_acc must have an integer value')
+                elif not 0 <= gender_acc <= 100:
+                    raise WrappedValueError('gender accuracy is not in range (0,100)')
```

Comment:
I would use `gender_acc` instead of `gender accuracy`. Maybe `gender_acc (gender_accuracy)`